orange/src/test/java/com/hartwig/hmftools/orange/algo/cuppa/CuppaDataFactoryTest.java

```diff
@@ -12,22 +14,114 @@
 import com.hartwig.hmftools.datamodel.cuppa.CuppaPrediction;
 import com.hartwig.hmftools.datamodel.cuppa.ImmutableCuppaPrediction;
 
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
 import org.junit.Test;
 
 public class CuppaDataFactoryTest
 {
     private static final double EPSILON = 1.0E-10;
 
-    private static final String CUPPA_VIS_DATA_TSV = Resources.getResource("test_run/cuppa/tumor_sample.cuppa.vis_data.tsv").getPath();
+    private static final String CUPPA_VIS_DATA_WITH_RNA_TSV = Resources.getResource("cuppa/with_rna.vis_data.tsv").getPath();
+    private static final String CUPPA_VIS_DATA_WITHOUT_RNA_TSV =
+            Resources.getResource("test_run/cuppa/tumor_sample.cuppa.vis_data.tsv").getPath();
 
     @Test
-    public void canExtractPredictionsFromCuppaV2() throws Exception
+    public void canExtractPredictionsFromCuppaV2IncludingRna() throws Exception
```

Comment:
Would remove the reference to CUPPA version in code. 

When we test a function related to purple we also don't use "purpleV3"!

Reply:
Agreed